recipes/plumed/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "plumed" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/plumed/plumed2/archive/v{{ version }}.tar.gz
+  sha256: de309980dcfd6f6e0e70e138856f4bd9eb4d8a513906a5e6389f18a5af7f2eba
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+# Most programs should link libplumedWrapper.a.
+# They will be forward/backward compatible with all plumed 2 releases.
+    - {{ pin_subpackage('plumed', min_pin='x', max_pin='x') }}
+# Programs linking libplumed.so should pin to min_pin='x.x', max_pin='x'.
+# Programs linking libplumedKernel.so should pin to min_pin='x.x.x'. max_pin='x.x.x'.
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+# xxd is provided by package vim and used during build
+    - vim  # [linux]
+  host:
+    - gsl
+    - libblas
+    - liblapack
+    - xdrfile
+    - zlib
+
+test:
+  commands:
+    - plumed info --root
+    - plumed info --long-version
+    - plumed-patch -h
+    - plumed config has external_blas external_lapack gsl xdrfile zlib
+    - $PREFIX/lib/plumed/plumed-runtime -h
+    - test -f $PREFIX/lib/libplumed.a
+    - test -f $PREFIX/lib/libplumed$SHLIB_EXT
+    - test -f $PREFIX/lib/libplumedWrapper.a
+    - test -f $PREFIX/lib/libplumedKernel$SHLIB_EXT
+    - conda inspect linkages -p $PREFIX $PKG_NAME
+    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
+
+about:
+  home: http://www.plumed.org/
+  license: LGPL-3.0
```

Comment:
```suggestion
  license: LGPL-3.0
  license_family: GPL
```

Reply:
I guess it can be `license_family: LGPL`

Reply:
The LGPL licenses are part of the GPL family.